tests/env_file_test.conf

```diff
@@ -1,3 +1,4 @@
+# Ignored comment
 VAR_OTHER=file
-
+# Comment
```

Comment:
If I remember correctly, it is because systemd accept `;` as prefix for a commented line.